lib/src/model/movies_result_model.dart

```diff
@@ -0,0 +1,65 @@
+class MoviesResultModel {
```

Comment:
these file and class are already on the model folder, you should rename it to MovieResult or something similar

Reply:
done